bin/build_comgr.sh

```diff
@@ -70,23 +76,26 @@ if [ "$1" != "nocmake" ] && [ "$1" != "install" ] ; then
 
    DEVICELIBS_BUILD_PATH=$AOMP_REPOS/build/AOMP_LIBDEVICE_REPO_NAME
    PACKAGE_ROOT=$AOMP_REPOS/$AOMP_COMGR_REPO_NAME/lib/comgr
-   ${AOMP_CMAKE} \
-      -DCMAKE_PREFIX_PATH="$AOMP/include/amd_comgr;$AOMP/lib/cmake;$DEVICELIBS_BUILD_PATH;$PACKAGE_ROOT" \
-      -DCMAKE_INSTALL_PREFIX=$INSTALL_COMGR \
-      -DCMAKE_INSTALL_LIBDIR=lib \
-      -DCMAKE_BUILD_TYPE=$BUILDTYPE \
-      -DBUILD_TESTING=OFF \
-      $AOMP_ORIGIN_RPATH \
-      -DROCM_DIR=$AOMP_INSTALL_DIR  \
-      -DLLVM_DIR=$AOMP_INSTALL_DIR \
-      -DClang_DIR=$AOMP_INSTALL_DIR \
-      $AOMP_REPOS/$AOMP_COMGR_REPO_NAME/lib/comgr
-
+	 MYCMAKEOPTS="-DCMAKE_PREFIX_PATH='$AOMP/include/amd_comgr;$AOMP/lib/cmake;$DEVICELIBS_BUILD_PATH;$PACKAGE_ROOT' -DCMAKE_INSTALL_PREFIX='$INSTALL_COMGR' -DCMAKE_BUILD_TYPE=$BUILDTYPE -DBUILD_TESTING=OFF $AOMP_ORIGIN_RPATH -DROCM_DIR=$AOMP_INSTALL_DIR -DLLVM_DIR=$AOMP_INSTALL_DIR -DClang_DIR=$AOMP_INSTALL_DIR"
```

Comment:
could you reformat this similar to code it replaces ?  easier to compare

Reply:
Creating another pull request with the reformatted version